src/material/core/theming/_check-duplicate-styles.scss

```diff
@@ -0,0 +1,69 @@
+@import './theming';
+
+// Whether duplication warnings should be disabled. Warnings enabled by default.
+$mat-theme-ignore-duplication-warnings: false !default;
```

Comment:
Rather than making this a variable someone would override, could we make it a function like `disable-duplicate-theme-styles-warning()`?

Reply:
I considered that, but I thought it won't be that convenient since Sass doesn't allow us to call functions without assigning the return value to a variable. e.g. folks would need to do:

```scss
$tmp-var: disable-duplicate-theme-styles-warning();
```

Unless I miss something here, I think the variable is the better choice.